### ISSUE TITLE
Trino JDBC Create table throw Exception

### ISSUE THREAD

**Issue by hqbhoho** [trinodb/trino#7368]
I use trino jdbc create table will throw exception. But create table is success；
![image](https://user-images.githubusercontent.com/37388552/111861719-62718b80-898b-11eb-9054-be60d416d348.png)

java.sql.SQLException: Error executing query
	at io.trino.jdbc.TrinoStatement.internalExecute(TrinoStatement.java:274)
	at io.trino.jdbc.TrinoStatement.execute(TrinoStatement.java:227)
	at org.lilith.bigdata.uds.data.exec.runner.PrestoSqlRunner.runQuery(PrestoSqlRunner.java:32)
	at org.lilith.bigdata.uds.data.TestJDBC.testPresto(TestJDBC.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:230)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:58)
Caused by: java.lang.IllegalStateException: current position is still valid
	at io.trino.jdbc.$internal.guava.base.Preconditions.checkState(Preconditions.java:508)
	at io.trino.jdbc.$internal.client.StatementClientV1.finalStatusInfo(StatementClientV1.java:246)
	at io.trino.jdbc.TrinoStatement.internalExecute(TrinoStatement.java:264)
	... 28 more

**Comment by ebyhr** [comment]
I couldn't understand the issue. Could you rephrase and share the steps to reproduce including Trino version?
